### PR TITLE
fix(components): [table] not scroll empty block when scrolling table

### PR DESCRIPTION
**Pull request by plainheart**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

I'm not sure if this behavior is by design but I always feel that the empty shouldn't be placed in the scroll view as it also unexpectedly scrolls together with the content when the table is scrollable. See the comparison below,

| Current | After |
| :----: | :----: |
| current behavior | expected behavior |

Playground is available [here](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICAgIDxlbC10YWJsZSBib3JkZXIgc3R5bGU9XCJ3aWR0aDogNTAwcHhcIj5cbiAgICAgIDxlbC10YWJsZS1jb2x1bW4gcHJvcD1cImRhdGVcIiBsYWJlbD1cIkRhdGVcIiB3aWR0aD1cIjE4MFwiIC8+XG4gICAgICA8ZWwtdGFibGUtY29sdW1uIHByb3A9XCJuYW1lXCIgbGFiZWw9XCJOYW1lXCIgd2lkdGg9XCIyMDBcIiAvPlxuICAgICAgPGVsLXRhYmxlLWNvbHVtbiBwcm9wPVwiYWRkcmVzc1wiIGxhYmVsPVwiQWRkcmVzc1wiIHdpZHRoPVwiNDAwXCIgLz5cbiAgICA8L2VsLXRhYmxlPlxuPC90ZW1wbGF0ZT5cblxuPHNjcmlwdCBsYW5nPVwidHNcIiBzZXR1cD5cblxuPC9zY3JpcHQ+XG5cbjxzdHlsZT5cbmh0bWwsIGJvZHkge1xuICB3aWR0aDogMTAwdnc7XG4gIGhlaWdodDogMTAwdmg7XG4gIG1hcmdpbjogMDtcbn1cbiNhcHAge1xuICBoZWlnaHQ6IDEwMCU7XG4gIGRpc3BsYXk6IGZsZXg7XG4gIGFsaWduLWl0ZW1zOiBjZW50ZXI7XG4gIGp1c3RpZnktY29udGVudDogY2VudGVyO1xuICBmbGV4LWRpcmVjdGlvbjogY29sdW1uO1xufVxuPC9zdHlsZT5cblxuXG5cbiIsImltcG9ydC1tYXAuanNvbiI6IntcImltcG9ydHNcIjp7fX0ifQ==).

### CODE REVIEW

**Comment by JeremyWuuuuu** [comment]
@plainheart  looks like you didn't update the tests.

**Comment by plainheart** [comment]
I'm waiting for the response about if this behavior is by design, thus no changes will be applied to the test cases. And if this change is accepted, this broken test case can be removed in fact.

**Comment by msidolphin** [comment]
Could you please add a demo for this🙏? so that we can discuss with the designer

**Comment by plainheart** [comment]
@msidolphin Thank you for your response. There is a simple playground url for this behavior in the bottom of the PR description.

**Comment by msidolphin** [comment]
> @msidolphin Thank you for your response. There is a simple playground url for this behavior in the bottom of the PR description.

I want to verify the changes in the document

**Comment by plainheart** [comment]
@msidolphin How do you hope where the demo should be added?

**Comment by msidolphin** [comment]
> @msidolphin What do you think where the demo should be added?

No need to add it, I found a problem, when the cursor is in the empty area, it can't trigger horizontal scrolling. my suggestion is to implement it with sticky,  empty area is still in the scroll-view

**Comment by plainheart** [comment]
Good catch!

**Comment by plainheart** [comment]
@msidolphin Fixed that scroll event can't be triggered on the empty block. Now it's ready for your one more review.

File: packages/components/table/src/table/style-helper.ts [hunk 1]
```diff
@@ -43,8 +43,7 @@ function useStyle<T>(
   })
   const isGroup = ref(false)
   const scrollbarViewStyle = {
-    display: 'inline-flex',
-    flexDirection: 'column',
+    display: 'inline-block',
```
Comment (msidolphin) [inline packages/components/table/src/table/style-helper.ts:46]: Please do not change here, inline-block will make the wrong height of scroll-view

Comment (plainheart) [reply packages/components/table/src/table/style-helper.ts]: This is to fix an unexpected gap when using `inline-flex` & `flex-direction: column`. If you have some better idea to fix the gap, then we can revert this change.

Comment (msidolphin) [reply packages/components/table/src/table/style-helper.ts:46]: > This is to fix an unexpected gap when using `inline-flex` & `flex-direction: column`. If you have some better idea to fix the gap, then we can revert this change.
> 
> 

It seems to have solved the problem 👍

**Review by msidolphin — APPROVED** [review]

**Comment by plainheart** [comment]
@msidolphin Could you take some time to take a look at the changes in #6694? In our projects, we usually define many columns and use the `min-width` to ensure it has a proper width to show the content. Unfortunately, this fix didn't solve such a case so the final result is still not perfect. And I saw another developer reported the same concern as well. Thanks a lot!

**Pull request merged**